study.md

```diff
@@ -34,7 +34,12 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+Array - map { |obj| block } → array
+Enumerator - map → an_enumerator If there is no block it returns the given enumerator
+
+Array - zip(arg, ...) → new_ary
+Enumerator - zip(arg, ...) { |arr| block } → nil Eventhough the enum number is more than array length it will generate as much as enum items but rest will be nul
+
```

Comment:
This does not answer the question.

`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.